dev/com.ibm.ws.security.oidc.client_fat/publish/servers/com.ibm.ws.security.openidconnect.client-1.0_fat.rp/configs/rp_server_withOpStub_sigAlg.xml

```diff
@@ -0,0 +1,257 @@
+<!--
+    Copyright (c) 2021 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+
+<server>
+
+	<include location="${server.config.dir}/imports/oidcClientFeatures.xml" />
+
+	<include location="${server.config.dir}/imports/goodBasicRegistry.xml" />
+
+	<include location="${server.config.dir}/imports/goodSSLSettings.xml" />
+
+	<include location="${server.config.dir}/imports/formlogin_1.xml" />
+
+	<include location="${server.config.dir}/imports/rp_fatTestPorts.xml" />
+
+    <keyStore
+         id="trust_allSigAlg"
+         password="Liberty"
+         type="PKCS12"
+         location="${server.config.dir}/allAlgTrustStore.p12" />
+
+    <keyStore
+         id="key_allSigAlg"
+         password="Liberty"
+         type="PKCS12"
+         location="${server.config.dir}/allAlgKeyStore.p12" />
+
+	<authFilter id="authFilterHS256">
+		<requestUrl
+			id="requestUrlHS256"
+			urlPattern="/simple/HS256"
+			matchType="contains" />
+	</authFilter>
+
+	<openidConnectClient
+		id="hs256"
+		scope="openid profile"
+		clientId="client01"
+		clientSecret="{xor}LDo8LTor"
+		sharedKey="secret"
+		mapIdentityToRegistryUser="true"
+		httpsRequired="false"
+		redirectToRPHostAndPort="http://localhost:${bvt.prop.security_2_HTTP_default}"
+		authorizationEndpointUrl="http://localhost:${bvt.prop.security_1_HTTP_default}/oidc/endpoint/OidcConfigSample/authorize"
+		tokenEndpointUrl="http://localhost:${bvt.prop.security_1_HTTP_default}/TokenEndpointServlet/getToken"  
+		validateAccessTokenLocally="true"
+		authFilterRef="authFilterHS256"
+	>
+	</openidConnectClient>
+
+	<authFilter id="authFilterHS384">
+		<requestUrl
+			id="requestUrlHS384"
+			urlPattern="/simple/HS384"
+			matchType="contains" />
+	</authFilter>
+
+	<openidConnectClient
+		id="hs384"
```

Comment:
Might need `signatureAlgorithm` added for the right algorithm (same for the HS512 config).

Reply:
I didn't spend much time on new HS algorithms - the issue we have for new signature algs focused on ES and RS - wasn't spending much time on HS until I knew if we were going to support them
Will update as appropriate when I get the runtime code